packages/ruby_rubocop.rb

```diff
@@ -51,8 +46,11 @@ def self.postinstall
 
     puts "Installing Chromebrew rubocop config file at #{@xdg_config_home}/rubocop/config.yml".lightblue
     puts 'This can be overridden by a ~/.rubocop.yml'.lightblue
-    FileUtils.mkdir_p "#{@xdg_config_home}/rubocop"
-    downloader 'https://github.com/chromebrew/chromebrew/raw/master/.rubocop.yml', 'SKIP', "#{@xdg_config_home}/rubocop/config.yml"
+    # FileUtils.mkdir_p "#{@xdg_config_home}/rubocop"
+    # downloader 'https://github.com/chromebrew/chromebrew/raw/master/.rubocop.yml', 'SKIP',
+    #            "#{@xdg_config_home}/rubocop/config.yml"
+    # Prevent breakage on ruby upgrades by not invoking ruby extensions during postinstall.
+    system "curl -Ls https://github.com/chromebrew/chromebrew/raw/master/.rubocop.yml -o #{@xdg_config_home}/rubocop/config.yml"
```

Comment:
Isn't this already in the `def self.install` section?

Reply:
Would it make sense to use `CREW_DEST_HOME`?

Reply:
Using `CREW_DEST_HOME` will ensure the config file is included in the package and is deleted with the package is removed.

Reply:
The postinstall mentions that CREW_DEST_HOME is a place where one can have a custom file which will override the crew version of the config file, so I don't think we should be putting the crew rubocop config file there.